frontend/scss/base.scss

```diff
@@ -11,14 +11,18 @@
 html, body {
   padding: 0;
   margin: 0;
-  min-width: 320px;
+  min-width: 240px;
```

Comment:
Can we extract the 240px into a variable?

Reply:
Good point. Done